src/components/web3/api.ts

```diff
@@ -13,7 +14,7 @@ const SIMPLEHASH_PROXY_ROOT_URL = "/api/v1/simplehash";
 export interface Web3Authentication {
   method: string;
   proof: {
-    signer: string;
+    signer: any;
```

Comment:
nit: What's the value in changing this to an any type? Seems like keeping it as a string would be more useful here.

Reply:
fixed it. Back to `string`